README.md

```diff
@@ -70,6 +82,10 @@ Example Playbooks
       log_level: INFO
       apm_enabled: "true" # has to be set as a string
       logs_enabled: true  # log collection is available on agent 6
+      process_config: # on agent 6
+        enabled: "true" # to enable process-agent
```

Comment:
Maybe say `has to be set as a string. possible values are: "true", "false" (for only container collection), "disabled" to disable the process-agent entirely`